parsec-evidence-cca.md

```diff
@@ -0,0 +1,71 @@
+# CCA Parsec Evidence
+
+This document describes the format of the credential produced by the Parsec
+service as a key attestation credential for applications running in an Arm CCA realm.
+
+The Parsec service produces a blob of binary data as a response to a key
+attestation operation issued by a client. The service takes in a nonce and uses
+it as freshness for the attestation token.
+
+The attestation token produced by Parsec is a CBOR encoded structure, namely a
+[Combined Attestation Bundle
+(CAB)](https://datatracker.ietf.org/doc/draft-bft-rats-kat/). The format is 
+described below in CDDL.
+
+```
+parsecCcaAttestation = {
+    "kat" => UCCS-KAT,
+    "pat" => CCA-token
+}
+
+UCCS-KAT = <TBD601>({
+  &(eat_nonce: 10) => bstr .size (8..64)
+  &(cnf: 8) => ik-pub
+})
+
+ik-pub = cnf-map
+
+cnf-map = {
+  &(cose-key: 1) => COSE_Key
+}
+```
+
+- `pat`: An Arm CCA Attestation Token as defined in the [Arm CCA Realm Management Monitor specification](https://developer.arm.com/documentation/den0137/latest) (see "Attestation Token Format" section).
+- `kat`: A sidecar token used to link the identity key of the application to the Platform Attestation Token (PAT) contained in `pat`.
```

Comment:
As mentioned above, the Line 34 creates the confusion and ambiguity.

Reply:
We can re-word it as:

`kat`: A sidecar token used to link the identity key of the application to the Platform Attestation Token (PAT) identified by  `pat`.